$OpenBSD: patch-src_main_c,v 1.3 2009/11/06 16:57:23 edd Exp $

set some sane defaults:

use mplayer's software volume control (see pkg/MESSAGE)
disable bar animation in full screen

fix from upstream svn (update volume control)

--- src/main.c.orig	Thu Sep 17 09:56:06 2009
+++ src/main.c	Fri Nov  6 05:22:16 2009
@@ -422,27 +422,13 @@ gint play_iter(GtkTreeIter * playiter, gint restart_se
         g_strlcpy(thread_data->subtitle, subtitle, 1024);
         g_free(subtitle);
     }
-    /*
-       #ifdef HAVE_ASOUNDLIB
-       if (!softvol && ao != NULL
-       && (g_ascii_strcasecmp(ao, "alsa") == 0
-       || (use_pulse_flat_volume && g_ascii_strcasecmp(ao, "pulse") == 0))) {
-       volume = (gint) get_alsa_volume(TRUE);
-       idledata->volume = volume;
-       #if GTK2_12_ENABLED
-       gtk_scale_button_set_value(GTK_SCALE_BUTTON(vol_slider), volume);
-       #else
-       gtk_range_set_value(GTK_RANGE(vol_slider), volume);
-       #endif
-       }
-       #endif
 
-       #if GTK2_12_ENABLED
-       volume = gtk_scale_button_get_value(GTK_SCALE_BUTTON(vol_slider));
-       #else
-       volume = gtk_range_get_value(GTK_RANGE(vol_slider));
-       #endif
-     */
+#if GTK2_12_ENABLED
+    volume = gtk_scale_button_get_value(GTK_SCALE_BUTTON(vol_slider));
+#else
+    volume = gtk_range_get_value(GTK_RANGE(vol_slider));
+#endif
+
     if (g_ascii_strcasecmp(thread_data->filename, "") != 0) {
         if (!device_name(thread_data->filename) && !streaming_media(thread_data->filename)) {
             if (!g_file_test(thread_data->filename, G_FILE_TEST_EXISTS)) {
@@ -699,7 +685,7 @@ int main(int argc, char *argv[])
     playlist_visible = FALSE;
     disable_fullscreen = FALSE;
     disable_framedrop = FALSE;
-    softvol = FALSE;
+    softvol = TRUE;
     subtitlefont = NULL;
     subtitle_codepage = NULL;
     subtitle_color = NULL;
@@ -711,7 +697,7 @@ int main(int argc, char *argv[])
     reallyverbose = 0;
     embedding_disabled = FALSE;
     disable_pause_on_click = FALSE;
-    disable_animation = FALSE;
+    disable_animation = TRUE;
     auto_hide_timeout = 3;
     use_mediakeys = TRUE;
     use_defaultpl = FALSE;
